ansible/scripts/deploy-cluster.sh

```diff
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-inventory=${INVENTORY:-inventory}
+. ./init.sh
 
-ansible-playbook -i ${inventory} deploy-cluster.yml $@
+inventory=${INVENTORY:-${INVENTORY_DIR}/inventory}
```

Comment:
Currently, the script does not do anything at all except it points to kubernetes default inventory and roles directories. In future, it can be altered and extended accordingly.

The main point of the scripts directory is to provide common ground for other scripts so there is no need for additional settings. This is a step towards re-usability of playbooks.

E.g. what I do in my CI is to copy dynamic inventory script under ${INVENTORY_DIR} so it is at the same directory as ``group_vars``.